routes/grading-url.js

```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var router = express.Router();
+
+/* retrieve assignment metadata for a course */
+router.route('/getUrlToGrade/?assignment=:assignment_id')   // not sure if this is a valid way to do this
```

Comment:
It's either going to be:

    router.get('/getUrlToGrade', function(req, res) {
        var assignment_id = req.query.assignment;

or

    router.get('/getUrlToGrade/:assignment', function(req, res) {
        var assignment_id = req.param.assignment;